Vipul Kumar/if else/11_upper_and_lower.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    char x;
+    cin >> x;
+    int ascii = int(x);
```

Comment:
you don't need to convert it explicitly it will converted by the compiler itself, you can directly do `if(x >= 'A' && x <= 'Z')`